rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -257,20 +236,24 @@ def set_state(self, state: Dict[str, Any]) -> None:
                 obtained by calling `self.get_state()`.
         """
         # The actual storage.
-        self._storage = state["_storage"]
-        self._next_idx = state["_next_idx"]
+        self._storage_unit = state["_storage_unit"]
+        self._storage_location = state["_storage_location"]
+        self._storage = self._create_storage(1)
```

Comment:
I think we should add a utility "storage.from_state()" method later.
Users want to start offline algorithms with some hand-crafted buffer sometimes and this would come in handy here.

Reply:
Yes, this might make things easier :+1: